examples/java/unit-test/rest/TwilioRestTest.java

```diff
@@ -412,4 +412,12 @@ public void testShouldSupportDeserializationTypes() {
         assertEquals("USD", account.iterator().next().getPriceUnit().toString() );
         assertEquals("2021-03-23T21:43:32.010069453Z[UTC]",account.iterator().next().getTestDateTime().toString());
     }
+
+    @Test
+    public void testRequestNonStandardDomainName() {
```

Comment:
Improved this testcase [here](https://github.com/twilio/twilio-oai-generator/blob/40156c24a43fdbd49b159a11c75074b05c4a892a/examples/java/unit-test/rest/TwilioRestTest.java#L436)